runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -1101,6 +1108,58 @@ TR::CompilationInfoPerThreadRemote::processEntry(TR_MethodToBeCompiled &entry, J
       stream->writeError(compilationLowPhysicalMemory, (uint64_t) computeServerMemoryState(getCompilationInfo()));
       abortCompilation = true;
       }
+   catch (const JITServer::StreamAotCacheMapRequest &e)
+      {
+      const std::string& aotCacheName = e.getCacheName();
+
+      if (TR::Options::getVerboseOption(TR_VerboseJITServer))
+         {
+         TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer,
+            "compThreadID=%d handling request for AOT cache %s method list",
+            getCompThreadId(), aotCacheName.c_str());
+         }
+
+      auto aotCacheMap = compInfo->getJITServerAOTCacheMap();
+      bool pending = false;
+      auto aotCache = aotCacheMap->get(aotCacheName, 0, pending);
```

Comment:
Ask Christian about the interaction of this code with AOT cache persistence.
`@param pending Output flag indicating that a cache load operation from file is in progress, but not read yet`
Can we just ignore the `pending` parameter?